webextensions/api/declarativeNetRequest.json

```diff
@@ -0,0 +1,1656 @@
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/URLTransform",
+            "support": {
+              "chrome": {
+                "version_added": "84"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "queryTransform": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "84"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            },
+            "addOrReplaceParams": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "84"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "15"
+                  },
+                  "safari_ios": "mirror"
+                }
+              },
+              "replaceOnly": {
+                "__compat": {
+                  "support": {
+                    "chrome": {
+                      "version_added": "94"
+                    },
+                    "edge": "mirror",
+                    "firefox": {
+                      "version_added": false
+                    },
+                    "firefox_android": "mirror",
+                    "opera": "mirror",
+                    "safari": {
+                      "version_added": "15"
+                    },
+                    "safari_ios": "mirror"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "getAvailableStaticRuleCount": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/getAvailableStaticRuleCount",
+            "support": {
+              "chrome": {
+                "version_added": "89"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
```

Comment:
`getAvailableStaticRuleCount` is not supported by Safari.

Reply:
```suggestion
                "version_added": false
```